### PR TITLE
Removes Advance release grenades

### PR DESCRIPTION
**Pull request by shazbot194**
So this PR was given to me from Spartan who was given it by Crazylemon. This is the description he gave.

"Since many reactions do not scale based on reagent volume, the advanced release grenade's behavior is currently poorly balanced. As such, the design is now removed from what R&D can research until reactions are better-balanced for repeat controlled use"

This PR just removes it from the protolathe, but keeps the item in game. 

:cl:
rscdel: R&D can no longer create advanced-release grenades due to certain reactions not scaling with reagent amount
/:cl:

### CODE REVIEW

**Comment by bigfatbananacyclops** [comment]
Examples of poor balance? i see no issue with it, its fun to interact with, there hasn't been any people i've seen with a issue about it. removing it is just removing something fun that people could interact with, sure it can be used with chemicals for some sort of self defense, but there's stuff to prevent that from hurting you. It should stay as long as it pleases everyone.

**Comment by Kyep** [comment]
Are these even used?

I only remember seeing these used one time, and, that time, admins saw the usage but (IIRC) did not consider it a problem.

I've heard of them being used one additional time, but even then it wasn't something super-destructive. It was (if memory serves) mostly dark matter, a pull/push effect. Yeah, annoying, but if memory serves, that has a maximum range, meaning you can simply shoot the person from outside of that range. 

Has there been some great spree of people using these in overpowered ways that I've just had the good luck to never see?

Or are we mostly removing this out of fear of ways that they might potentially be used, even though such usage hasn't actually happened?

If we are removing these based on fear of what might potentially happen, but has not happened yet in the... long time... since they were added... are the potentials actually that bad?

I play science a fair bit, and the reactions that come to mind, off the top of my head, which do not scale based on reagent volume, are things like explosions and fireballs. Carrying something on your person that continually sets you and the surrounding area on fire, or causes a small explosion, doesn't seem that OP. It seems more like a very quick way to find yourself dead, and subject to a DNR order.

Is there some reaction that is massively advantageous to the user, can be repeated endlessly on your person without negative consequence for you, and doesn't have another easy counter like 'shoot them from a few tiles away'? Has anyone actually been using such a reaction in a grenade?
I'm curious.

If you don't want to post the actual name of the reaction here, at least message it to a maint/headmin. If we're going to remove these, we should at least do so on the basis of a concrete problem, rather than a vague fear about there possibly potentially being un-named reactions that might maybe be a problem.

**Comment by FalseIncarnate** [comment]
These are used, and were used recently which brought this matter to light within the staff.

Since reagent effects do not consistently scale with volume, this led to the request from within staff to have this temporarily removed from player access until the reagents can be rebalanced. This is not an outright removal of the item and the intent is that it will be returned to player access once reactions are updated to better scale with volume.

Some effects like Sorium don't currently scale at all with reagent volume (if any is present, the effect occurs at full power) and offer little to no risk to the user (epicenter of the effect). With how the advanced release grenade works, this means you have a multi-use, low-risk, and somewhat spammable tool on your hands with practically no counter.

TL;DR: This is not a full removal, it will return as obtainable once chems get rebalanced. This was prompted by recent events discussed by staff and the PR is requested by a maintainer. Just because you don't see the problem doesn't mean it shouldn't be fixed.

**Comment by DarkPyrolord** [comment]
Alright, this PR is going to be closed.

The reason as to why is because for this sort of issue the problem reagents should be the ones modified to at least have some form of scaling or lesser reaction when it comes to this issue, not the grenade itself as this can still easily be done (but in a slower manner) if someone just grabs a dropper and three beakers.
